.github/workflows/zcashd-manual-deploy.yml

```
@@ -64,6 +64,7 @@ jobs:
           --machine-type ${{ env.MACHINE_TYPE }} \
           --service-account ${{ env.DEPLOY_SA }} \
           --scopes cloud-platform \
+          --labels=app=zcashd,environment=qa,github_ref=${{ env.GITHUB_REF_SLUG_URL }} \
```

I think this workflow is used to deploy `zcashd` instances to production, but I could be wrong.

```suggestion
          --labels=app=zcashd,environment=prod,github_ref=${{ env.GITHUB_REF_SLUG_URL }} \
```

---

I think this depends if we're meant to keep `zcashd` instances running on our infrastructure.

Are we running those instances _because we **need them**_, or are we running those instances _for **testing** and or **qa** purposes_?

cc: @mpguerra @dconnolly

---

We're not currently running any zcashd instances at all.

But we previously used it to:
- generate Zebra checkpoints

And we planned on doing it to:
- help testnet stability